modules/core/src/main/scala/org/tessellation/cli/method.scala

```diff
@@ -74,6 +74,8 @@ object method {
 
     val l0SeedlistPath = seedlistPath
 
+    val prioritySeedlistPath = None
```

Comment:
can you explain briefly what is a priority seedlist and how what is the difference between the original seedlist? also - since we already extended the original seedlist, can't it be one more flag added to the seedlist rather than a separate thing?

Reply:
@marcinwadon 

My understanding is that the priority seedlist should be comprised of GL0 nodes. Per the Jira ticket, it is used to check for majority and see if the current node is aligned with it.

cc: @ryle-goehausen

Reply:
what happens if nodes from the priority seedlist are misaligned?

Reply:
Is this the same or derived from the allowanceList on GL0?

Reply:
Yeah, so IMO it makes sense to provide another field to the existing seedlist and mark these nodes.

Reply:
Can you rephrase what you mean by schema changes to resolve forking?

Reply:
hey @sdavidp, what about this additional field instead of separate list?

Reply:
I thought @ryle-goehausen already addressed this. AFAIK he still wants it to be separate from the regular seed list.

Reply:
ok, I didn't see anything about separate list vs additional field marking peers. as far as I see Ryle just mentioned it should be a subset of the seedlist, but technically it can be achieved in many ways. adding additional field to the seedlist would mitigate other issues like different peers on two seedlists, etc.
also, could you answer my question re: schema changes to resolve forking? 🤔

Reply:
I only wonder what in a situation where clients want different nodes to be the realign peers for their process - that would require adjusting the seedlist after downloading it.

Reply:
@marcinwadon I don't understand the question re. schema changes.

Reply:
@TheMMaciek regarding more unusual situations, for now I think we should just support two modes, one where it is provided and not provided, if it makes sense to add support for more operations we can add additional CLI parameters. Given that this should be replaced by scoring functions in the future (and the other user supplied scores list which chuong has already implemented,) I don't think it makes sense to adjust the schema. Since we'll have scores for each peer we should rely on adjustments to scores to determine this sort of behavior in the future. (And/or specific labels added to scoring CSV / new fields added there.) Adjusting the schema on the main list doesn't make as much sense for a specific feature we're using temporarily.

Reply:
@sdavidp @ryle-goehausen 
![image](https://github.com/Constellation-Labs/tessellation/assets/4959931/789d80cb-6a03-45af-914c-cbe67ff18570)

Reply:
So this is primarily intended (as we discussed in earlier meetings,) to prevent situation of submitting and synchronizing to incorrect GL0 fork, which will be exacerbated during updates, especially schema related ones or any situation where immediate upgrade forces a fork. The 'why' question of this PR we can discuss on JIRA ticket. We can resolve this conversation if there's no question here about the code itself to accomplish this.